src/helpers/figures/charts/scorecard_chart_config_builder.ts

```diff
@@ -318,7 +320,10 @@ class ScorecardChartConfigBuilder {
     );
 
     let isBaselineSplit = false;
-    if (baselineDescrFontSize < baselineValueFontSize / 2.5) {
+    if (
+      baselineDescrFontSize < baselineValueFontSize / 2.5 &&
+      this.baselineDescr.trim().includes(" ")
```

Comment:
why the "space" condition?

Reply:
To avoid making a false split when we validate the font size condition but we only have one word in the baselineDescr. Without this condition, we would try to split, resulting in a two lines description with the first line containing the word and the second line being empty.